app/src/main/java/com/epfl/drawyourpath/database/Database.kt

```diff
@@ -148,4 +166,89 @@ abstract class Database {
      * @return a future that indicate if the achievements of the user have been correctly updated.
      */
     abstract fun updateUserAchievements(userId: String, distanceDrawing: Double, activityTimeDrawing: Double): CompletableFuture<Unit>
+
+    /**
+     * Function used to create a chat conversation with other users of the DrawYourPath community.
+     * @param name name of the chat conversation
+     * @param membersList list of the members userId of the chat conversation(the creator must be included in this list)
+     * @param creatorId userId of the conversation creator
+     * @param welcomeMessage a welcome message in a string format
+     * @return a future that indicate if the conversation was correctly created inside the database
+     */
+    abstract fun createChatConversation(
+        name: String,
+        membersList: List<String>,
+        creatorId: String,
+        welcomeMessage: String,
+    ): CompletableFuture<Unit>
```

Comment:
Suggestion: Perhaps it would be useful if this function returns the id of the conversation.